README.rst

```diff
@@ -91,8 +91,7 @@ Contact Options
 
 - Feel free to use the `github3.py`_ tag on Stack Overflow for any questions
   you may have.
-- If you dislike Stack Overflow, it is preferred that you send an email to
-  github3.py@librelist.com .
+- If you dislike Stack Overflow, it is preferred that you [link to github discussions] .
```

Comment:
Should probably link to the discussions tab here for clarity, no?

Reply:
ah, now I see it :)